docs/resources/user.md

```diff
@@ -37,7 +37,7 @@ resource "redshift_user" "user_with_unrestricted_syslog" {
 - **connection_limit** (Number) The maximum number of database connections the user is permitted to have open concurrently. The limit isn't enforced for superusers.
 - **create_database** (Boolean) Allows the user to create new databases. By default user can't create new databases.
 - **id** (String) The ID of this resource.
-- **password** (String, Sensitive) Sets the user's password. Users can change their own passwords, unless the password is disabled. To disable password, omit this parameter or set it to `null`.
+- **password** (String, Sensitive) Sets the user's password. Users can change their own passwords, unless the password is disabled. To disable password, omit this parameter or set it to `null`.  This can also be a hashed password rather than the plaintext password.  Please refer to the [CREATE USER documentation](https://docs.aws.amazon.com/redshift/latest/dg/r_CREATE_USER.html) for information on creating a password hash.
```

Comment:
Please update the documentation at `password` field schema (https://github.com/brainly/terraform-provider-redshift/blob/master/redshift/resource_redshift_user.go#L84). The .md files are autogenerated from there.